Extension/package.json

```diff
@@ -1471,26 +1471,26 @@
     "watch": "webpack --watch --mode development"
   },
   "devDependencies": {
-    "@types/mocha": "^5.2.6",
-    "@types/node": "^12.0.0",
+    "@types/minimatch": "^3.0.3",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.0.10",
     "async-child-process": "^1.1.1",
     "await-notify": "^1.0.1",
     "gulp": "^4.0.2",
     "gulp-env": "^0.4.0",
     "gulp-mocha": "^6.0.0",
     "gulp-tslint": "^8.1.4",
-    "tslint": "^5.16.0",
-    "tslint-microsoft-contrib": "^6.1.1",
+    "ts-loader": "^6.0.4",
+    "tslint": "^5.18.0",
+    "tslint-microsoft-contrib": "^6.2.0",
     "tslint-no-unused-expression-chai": "^0.1.4",
-    "ts-loader": "^6.0.0",
-    "typescript": "^3.4.5",
+    "typescript": "^3.5.2",
     "vrsource-tslint-rules": "^6.0.0",
-    "vscode": "^1.1.34",
-    "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.2"
+    "vscode": "^1.1.35",
+    "webpack": "^4.35.2",
+    "webpack-cli": "^3.3.5"
   },
   "dependencies": {
-    "@types/minimatch": "^3.0.3",
```

Comment:
This was incorrectly being marked as a dependency, potentially causing our ThirdPartyNotices.txt to be incorrect (I'll update that later).

Reply:
I updated the ThirdPartyNotices.txt too.